modules/openapi-generator/src/main/resources/Java/libraries/jersey3/pom.mustache

```diff
@@ -408,7 +411,8 @@
         <jackson-version>2.17.1</jackson-version>
         <jackson-databind-version>2.17.1</jackson-databind-version>
         <jackson-databind-nullable-version>0.2.6</jackson-databind-nullable-version>
-        <jakarta-annotation-version>2.1.1</jakarta-annotation-version>
+        {{#useJakartaAnnotation}}
+        <jakarta-annotation-version>2.1.1</jakarta-annotation-version>{{/useJakartaAnnotation}}
```

Comment:
fyi. normally we put the start, end mustache tags in their own line (e.g. line 414) to avoid blank lines in the output